stf/src/sgx.rs

```diff
@@ -265,6 +265,23 @@ impl Stf {
                     Self::shield_funds(who, value)?;
                     Ok(())
                 }
+
+                TrustedCall::place_order(_main_account, _order, _proxy_account) => {
+                    // TODO implement and call into SGX runtime
```

Comment:
I don't think we will need that.

Reply:
I need to have an implementation there, otherwise the match is not exhaustive. Do I just remove the TODO then, or do you think a default clause _ => is better?

Reply:
I think for now a _ => would be for the best, leaving an empty ..::place_order or similiar there, suggests that there are / will be some use cases for it.

Reply:
done 👍